scheduler/scheduler-node/src/main/java/org/ow2/proactive/scheduler/task/executors/forked/env/ExecuteForkedTaskInsideNewJvm.java

```diff
@@ -50,7 +50,11 @@ private static TaskContext deserializeContext(String pathToFile) throws IOExcept
         try (ObjectInputStream inputStream = new ObjectInputStream(new FileInputStream(f))) {
             return (TaskContext) inputStream.readObject();
         } finally {
-            FileUtils.forceDelete(f);
+            try {
+                FileUtils.forceDelete(f);
+            } catch (Exception e) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Either log or rethrow this exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1166)